code/src/main/kotlin/com/expediagroup/sdk/core/authentication/strategy/AuthenticationStrategy.kt

```diff
@@ -1,13 +1,13 @@
-package com.expediagroup.sdk.v2.core.authentication.strategy
+package com.expediagroup.sdk.core.authentication.strategy
 
-import com.expediagroup.sdk.v2.core.trait.authentication.AuthenticationHandlerTrait
+import com.expediagroup.sdk.core.trait.authentication.AuthenticationHandlerTrait
 
 /**
  * Enumeration that represents different authentication strategies for a client configuration.
  *
  * @property handlerFactory A factory that creates an instance of the authentication handler
  *                          trait for the specific strategy.
  */
-enum class AuthenticationStrategy(val handlerFactory: AuthenticationHandlerTrait) {
+enum class AuthenticationStrategy(val handlerFactory: com.expediagroup.sdk.core.trait.authentication.AuthenticationHandlerTrait) {
```

Comment:
Why to use the full name? (applicable to other places too).

Reply:
Done.